examples/differential_rotation/comparing_rotation_models.py

```diff
@@ -0,0 +1,76 @@
+"""
+======================================
+Comparing differential-rotation models
+======================================
+
+How to compare differential-rotation models.
+
+The example uses the `~sunpy.coordinates.metaframes.RotatedSunFrame` coordinate
+metaframe in `sunpy.coordinates` to apply differential rotation to a coordinate.
+See :ref:`sunpy-coordinates-rotatedsunframe` for more details on using
+`~sunpy.coordinates.metaframes.RotatedSunFrame`.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import HeliographicStonyhurst, RotatedSunFrame
+from sunpy.data.sample import AIA_335_IMAGE
+from sunpy.sun.constants import sidereal_rotation_rate
+
+##############################################################################
+# First, we use an AIA observation primarily as a pretty background.  We also
+# define a coordinate rectangle with zero width in the longitude direction to
+# represent the meridian.
```

Comment:
```suggestion
# First, we use an AIA observation primarily as a pretty background.  We also
# define a line at zero degrees longitude to represent the meridian.
```
Seems simpler? Perhaps there's a reason for using rectangle instead?

Reply:
Ah, now I've made it to the end of the example I see why. I think it might be clearer to use an array of latitudes, and `plot_coord` at the end? Using `draw_quadrangle` is neat in this context, but I don't think very intuitive.

Reply:
We already have an example showing the use of `plot_coord()` and `RotatedSunFrame`, so I doubly like using `draw_quadrangle()` here instead.  I've revised the text, so take a look at the new description.